src/codec.ts

```diff
@@ -531,6 +550,7 @@ const TypeCode: {
   bool: 'BOOL',
   int64: 'INT64',
   pgOid: 'INT64',
+  float32: 'FLOAT32',
```

Comment:
Should also add documentation in "paramtype", refer https://github.com/googleapis/nodejs-spanner/blob/main/src/codec.ts#L570

Reply:
Done